app/api/ddi-index-api/cdo.js

```diff
@@ -24,7 +24,29 @@ const createCdo = async (cdo, user) => {
   return res
 }
 
+const getManageCdoDetails = async (indexNumber) => {
+  const payload = await get(`${cdoEndpoint}/${indexNumber}/manage`, options)
+  return payload
+}
+
+const getCdoTaskDetails = async (indexNumber, taskName) => {
+  const payload = await get(`${cdoEndpoint}/${indexNumber}/manage`, options)
+  return payload
+}
+
+const taskNameMapper = {
+  'send-application-pack': 'sendApplicationPack'
```

Comment:
Although this is a quick way of doing the lookup, probably better to use the generic-task-helper definition as the mapper so we don't duplicate the definitions in two places.